src/helpers/validation.ts

```diff
@@ -8,5 +8,5 @@ function isValidPhoneNumber(val: string) {
 export function customPhoneValidator(value: string): string[] {
   return isValidPhoneNumber(value)
     ? []
-    : [`Het telefoonnummer '${value}' lijkt geen valide telefoonnummer.`];
+    : [`'${value}' lijkt geen valide telefoonnummer.`];
```

Comment:
Why is this actually an array? Might be easier if it was a return type of `:string | undefined` and just return the error message if there's an validation error.

Reply:
Just for extendibility. At this time, we just use a single validation message. In the future, we might want to give more descriptive messages (for example, a message about the length, as well as a message about strange characters). By using an array, the contract doesn't have to change in that case. Then again, we don't need it right now. I'll think about it for a bit longer.